stripper/ze_tesv_skyrim_v5_6.cfg

```diff
@@ -1421,4 +1421,79 @@ filter:
 {
 	"classname" "infodecal"
 	"texture" "decals/decal_crater001a"
+}
+ 
+;Attempt to fix reported lags in certain areas due to missing sounds and Linux filesystem fuckery.
+;Though it might be an issue for Linux only, it won't hurt to remove the errors spammed in the console regardless.
```

Comment:
This change is fine to keep since the soundscript entries themselves do seem to be fucked (and some don't exist at all). But from checking logs of skyrim, it doesn't seem like bad soundscripts cause the missing sound errors in SoundLib2 that spam and drop performance on Linux. So I would adjust the comment to say this is only fixing bad sounds.